api/src/main/java/org/pmiops/workbench/db/model/DbInstitution.java

```diff
@@ -0,0 +1,95 @@
+package org.pmiops.workbench.db.model;
+
+import java.util.List;
+import javax.persistence.*;
+
+@Entity
+@Table(name = "institution")
+public class DbInstitution {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  @Column(name = "institution_id")
+  private long institutionId; // primary opaque key for DB use only
+
+  @Column(name = "api_id", nullable = false, unique = true)
+  private String apiId; // unique key exposed to API
+
+  @Column(name = "long_name", nullable = false)
+  private String longName;
+
+  @Column(name = "organization_type_enum")
```

Comment:
I wouldn't put the type in the column name. SQL will let you reflect that. Plus, if you change the type (like to a real SQL enum) later, you'd have to rename at the same time, which is a headache.

Reply:
Because the string+enum construction is a bit wacky, I want to make this explicit.

Reply:
+1 to explicitly differentiating it, though I still think it's redundant to put the type in the column name. Maybe name the enum `organization_type` and the text field `other_organization_type_description`?